tools/rosetta/CHANGELOG.md

```diff
@@ -36,6 +36,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### Improvements
+
+* (rosetta) [#13832](https://github.com/cosmos/cosmos-sdk/pull/13832) Correctly populates rosetta's `/network/status` endpoint response. Roseta data api is divided into its own go files (account, block, mempool, network).
```

Comment:
One last nit, sorry 😅, we don't need to prepend `rosetta` there.

Reply:
you're completely right